### ISSUE TITLE
Conversion between Celsius and Fahrenheit

### ISSUE THREAD

**Issue by andyhe886** [bmorelli25/ReactWeather#31]
Hi , is it possible to have both units of measurement for temperature atm?

**Comment by bmorelli25** [comment]
Unfortunately, it looks like that would require two API calls :/

**Comment by dosten** [comment]
Can't you use the well known formula to convert celsius to fahrenheit?

```
fahrenheit = celsius * (9/5) + 32 
```

```
celsius = (fahrenheit - 32) * (5/9) 
```

**Comment by bmorelli25** [comment]
Hah. Well duh, you totally could! Didn't think of that. Simple answers are often the best.

Okay, this seems like a pretty good enhancement: Convert temperature to Celsius and have a toggle (in the navbar maybe?) where users can toggle Celsius or Fahrenheit. 

Anyone want to take it on?

**Comment by andyhe886** [comment]
So, how does it work?,sry i'm new to this kind of open source project, if i want to make this enchancement, i fork & star and clone it. After i make the change. i just push it?

**Comment by ghost** [comment]
@andyhe886 Check the Code -> readme.md and in it you can find how to contribute for Hacktoberfest :)

By the way I can help you with the assignment. This is my simple API project with toggle button for changing 'C to 'F and vice versa. 

http://codepen.io/Dayfiri/full/mEjEEJ/

**Comment by bmorelli25** [comment]
@andyhe886, more or less, yes! After you've made the changes to your forked repository, you can go to my project and open a new "pull request". [Read this GitHub article on how Pull Requests work](https://help.github.com/articles/about-pull-requests/). This will let me know you're ready for your code to be merged into my project. If everything tests well on my end, I merge your code, and your name is added as a contributor to the project!

@Dayfiri - yes please! Your project seems to do exactly what I'm looking for. Feel free to give it a shot!

**Comment by ghost** [comment]
Ok, great. I'm on it :)

**Comment by andyhe886** [comment]
Alright :)

**Comment by bmorelli25** [comment]
Merged #45.
